### DISCUSSION TITLE
fork and branch

### DISCUSSION THREAD

**Discussion by huythanhpham138** [Schweinepriester/github-profile-achievements#64] [Q&A]
What is the difference between 'fork' and 'branch'?

**Comment by melvincwng** [comment]
@huythanh13082000,

A fork is simply just a duplicate copy of someone else's project/repository, whereas a branch is just another version of a repository.

Hope this clarifies.

**Reply by melvincwng** [reply]
@huythanh13082000, if this reply has helped you, please kindly 'Mark as answer'.

Thank you and have a great day ahead 😊

**Comment by ramesh2051** [comment]
A Fork is nothing but duplicate copy of Repository, and also Forks let you make changes to a project without affecting the original repository.

Branches are nothing but collection of commits and also branches are best used: when they are created as temporary places to work through a feature, with the intent to merge the branch with the origin. it is also a another version of a repository.

**Comment by devntv** [comment]
check documentation and you will understand all about it
link docs: https://docs.github.com/en/get-started/quickstart/fork-a-repo
